.github/02_performanceissue.md

```diff
@@ -0,0 +1,59 @@
+---
+name: Performance Issue
+about: Report a performance issue or regression.
+title: ''
+labels: performance
+---
+
+<!-- This is a template that helps us provide quicker feedback. Please use any relevant sections and delete anything you don't need. -->
+
+### Issue Description
+<!--
+Please include a clear and concise description of the problem.
+-->
+
+### Data
+<!--
+* Please include all information you've gathered about this performance issue. This includes:
+    -Timing
+    -Measurements
+    -Call stacks
+    -Dumps
+    -Regression?
+    -etc.
+* If possible please include text as text rather than images (so it shows up in searches).
+* If applicable please include before and after measurements.
+-->
```

Comment:
Regression could be its own section?

Reply:
I was wondering what the regression section would even ask for. I'll take inspiration from the docs templates entirely for this one.